src/pallavi/A12_ScannerProg2.java

```diff
@@ -0,0 +1,40 @@
+/*Assignment - 12 : 26th Dec'2021
+
+2. find max prime number from given range. [range : 2 to 100, ans ; 97]*/
+
+package pallavi;
+
+import java.util.Scanner;
+
+public class A12_ScannerProg2 {
+
+	void findMaxPrime(int startnum, int endnum) {
+		boolean isprime = false;
```

Comment:
this line should at line 15, as for every single number we need to init the variable to false;